01_Core/week02/submissions/index.html

```diff
@@ -0,0 +1,109 @@
+        </div>
+        <div class="career">
+            <h2 id="career">Career</h2>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+        </div>
+        <div class="form">
+            <h2 id="you">Tell me about yourself!</h2>
+        <form action="post" method="GET">
+            <div>
+                <label for="name">First Name:</label>
+                <input type="text" id="first-name">
+            </div>
+            <div>
+                <label for="name">Last Name:</label>
+                <input type="text" id="last-name">
+            </div>
+            <div>
+                <label for="gender">Gender:</label><br>
+                <input type="radio" name="gender" value="female" checked>Female<br>
+                <input type="radio" name="gender" value="male">Male<br>
+                <input type="radio" name="gender" value="other">Other
+            </div>
+            <div>
+                <label for="sport">What's your favorite sport?</label><br>
+                <select name="sport">
+                    <option value="basketball">Basketball</option>
+                    <option value="baseball">Baseball</option>
+                    <option value="football">Football</option>
+                    <option value="golf">Golf</option>
+                    <option value="tennis">Tennis</option>
+                </select>
+            </div>
+            <div>
+                <fieldset>
+                    <legend>What do you like on your pizza?</legend>
+                    <input type="checkbox" name="topping" value="pepperoni" id="pepperoni">
+                    <label for="pepperoni">Pepperoni</label>
+                    <input type="checkbox" name="topping" value="pineapple" id="pineapple">
+                    <label for="pineapple">Pineapple</label>
+                    <input type="checkbox" name="topping" value="bacon" id="bacon">
+                    <label for="bacon">Bacon</label>
+                    <input type="checkbox" name="topping" value="ham" id="ham">
+                    <label for="ham">Ham</label>
+                    <input type="checkbox" name="topping" value="sausage" id="sausage">
+                    <label for="sausage">Sausage</label>
+                </fieldset>
+            </div>
+            <button type="submit">Save</button>
+        </form>
+    </div>
+        <div class="bottom">
+            <h2>Click <a href="another-page.html">here</a> for a list of people who have already completed this survey.</h2>
```

Comment:
What you did isn't wrong, but try to make your a tags more descriptive than "here". Instead, you could do <a href="another-page.html">View a list of people who have already completed this survey</a>.